libyul/optimiser/NameDispenser.h

```diff
@@ -24,6 +24,7 @@
 #include <libyul/YulString.h>
 
 #include <set>
+#include <vector>
```

Comment:
You include ``vector``, but you *use* ``unordered_map`` below.
```suggestion
#include <unordered_map>
```